build.gradle

```diff
@@ -1,21 +1,33 @@
-plugins {
-    id 'java'
-    id 'org.jetbrains.intellij' version '0.3.5'
-}
 
 group 'io.vivy'
-version '1.0-SNAPSHOT'
+version '1.0'
 
-sourceCompatibility = 1.8
+allprojects {
 
-repositories {
-    mavenCentral()
-}
+    repositories {
+        mavenCentral()
+    }
 
-dependencies {
-    testCompile group: 'junit', name: 'junit', version: '4.12'
 }
 
-intellij {
-    version '2018.2.1'
-}
+
+subprojects {
+    apply plugin: 'java'
+
+    dependencies {
+
+        testImplementation 'org.junit.jupiter:junit-jupiter-api:5.2.0'
```

Comment:
please use https://github.com/spring-gradle-plugins/dependency-management-plugin
and junit bom, also put versions into dependency management section

Reply:
fixed